CHANGES.md

```diff
@@ -97,8 +105,7 @@
   Merlin has a concept of "typed holes" that are syntactically represented as `_`. Files
   that incorporate typed holes are not considered valid OCaml, but Merlin and OCaml-LSP
   support them. One example when such typed holes can occur is when on "destructs" a value,
-  e.g., destructing `(Some 1)` will generate code `match Some 1 with Some _ -> _ | None ->
-  _`. While the first underscore is a valid "match-all"/wildcard pattern, the rest of
+  e.g., destructing `(Some 1)` will generate code `match Some 1 with Some _ -> _ | None -> _`. While the first underscore is a valid "match-all"/wildcard pattern, the rest of
```

Comment:
What happened to the wrapping here?

Reply:
Markdown autoformatting at play here. Sorry, fixed.